targets/TARGET_Cypress/TARGET_PSOC6/COMPONENT_WHD/resources/nvram/TARGET_CYTFM_064B0S2_4343W/wifi_nvram_image.h

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright (c) 2019, Cypress Semiconductor Corporation, All Rights Reserved
+ * SPDX-License-Identifier: LicenseRef-PBL
```

Comment:
Have the correct additional files been provided with this license ? https://os.mbed.com/docs/mbed-os/v6.2/contributing/license.html#when-to-use-pbl

Reply:
Yes. Here - ./targets/TARGET_Cypress/TARGET_PSOC6/COMPONENT_WHD/resources/nvram/TARGET_CY8CKIT_064S1_4343W/wifi_nvram_image.h